docs/application/dotnet/samples/content/overview.md

```diff
@@ -90,6 +90,15 @@ for TD:
 			<p>This sample application demonstrates how to obtain and filter media content information from internal storages.</p>
 			</td>
 		</tr>
+		<tr>
+		<td>
+			<p><img alt="" height="225" src="media/tv26mimetypeconverter.png" width="400" /></p>
```

Comment:
I will discuss it with Doc part. It is OK with Internet Explorer, but, not OK with Chrome.

Reply:
Fine. Thank you.

Reply:
@Reni-90
Please never mind JavaScript. It is not for GitHub.

@annie-samsung 
If you have time, could you study JavaScript with your members?  When you take over Tizen Docs, it will help you to understand web pages.

Reply:
@journey2w 
Sure, will plan for this activity. Thanks for pointing it out.

Reply:
Squashed image will be fixed by site manager. not this page issue